ui/src/semantic-ui/site/elements/list.overrides

```diff
@@ -7,6 +7,11 @@
     width: 100%;
   }
 
+  .document-authors-list-height {
```

Comment:
please do not name classes containing css attribute name in the name of the class, if it is not only defining the attribute
in this case you name it `...-height` when it is also defining overflow and it might be confusing.
for this case, as it is very specific you could just use less parent selector:
```
.ui.list.document-authors-list{
... already defined stuff
&.expanded{
    max-height: 3em; <- also better use em unit in this case to keep consistency with the other text 
    overflow-y: scroll;
}
}